radicle/src/identity/project.rs

```diff
@@ -117,6 +117,19 @@ impl Doc<Verified> {
         false
     }
 
+    pub fn remove(&mut self, key: &crypto::PublicKey) -> Result<Option<Did>, DocError> {
```

Comment:
`undelegate`?

Reply:
`abdicate`? :eyes: Or other synonyms https://www.merriam-webster.com/thesaurus/abdicate I kind of like `rescind` too :smile:

Reply:
I'm ok with `rescind`, though we lose the symmetry with `delegate` :shrug:

Reply:
Well I actually looked up antonyms of delegate, as in "to delegate", and abdicate was what came up :) So the symmetry is there ;)